database/src/migrations/20230817000000_critter_observation_deployment_tables.ts

```diff
@@ -0,0 +1,184 @@
+import { Knex } from 'knex';
+
+/**
+ * Added critter and observation tables with trigger to check if critter instances is less than observation.total_count
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ *
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+
+  ----------------------------------------------------------------------------------------
+  -- Create Observation Table
+  ----------------------------------------------------------------------------------------
+
+  set search_path=biohub;
+
+  CREATE TABLE observation(
+    observation_id           integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    survey_id                integer           NOT NULL,
+    total_count   	         integer			     NOT NULL CHECK(total_count > 0),
```

Comment:
Nice

Reply:
Random thought: It does remind me of the existence of non-observation observations (ie: I specifically am recording the absence of an animal). But I think that is its own separate issue to tackle elsewhere/later, and isn't the same as a count of 0 anyways.